exercises/D-logical-operators/dist/exercise.dev.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+/*
+  Logical Operators
+  ---------------------------------
+  Using logical operators complete the unfinished statements.
+  The variables should have values that match the expected results.
+*/
+// Do not change these two statement
+var htmlLevel = 8;
+var cssLevel = 4; // Finish the statement to check whether HTML, CSS knowledge are above 5
+// (hint: use the comparison operator from before)
+
+var htmlLevelAbove5;
+var cssLevelAbove5;
+
+if (htmlLevel > 5) {
+  htmlLevelAbove5 = true;
+} else {
+  htmlLevelAbove5 = false;
+}
```

Comment:
Nice work! We could also use Conditional (ternary) operator to write simple boolean expression.
htmlLevelAbove5 = htmlLevel > 5 ? true : false ;